data/en/mbs_0823.json

```diff
@@ -0,0 +1,375 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "survey_id": "009",
+    "theme": "default",
+    "data_version": "0.0.1",
+    "title": "Monthly Business Survey",
+    "legal_basis": "StatisticsOfTradeAct",
+    "description": "mbs",
```

Comment:
should probably add the survey it is for description.

Reply:
Mmm - `"description": "mbs",` is consitent with all other MBS surveys so far

Reply:
in mine and i think in davids he puts the form type too, either way its not gonna make any difference really